02_bubble/shuutech/main.go

```diff
@@ -0,0 +1,16 @@
+package main
+
+import (
+	"fmt"
+	"sort"
+)
+
+func bubble(s []int) []int {
+	sort.Ints(s)
```

Comment:
This isn't an implementation of bubble sort, this is just calling a function from the standard library.

I believe the idea is to implement bubble sort yourself, and show your understanding of using arrays/slices in golang.

Reply:
I have updated main.go as per review. Please review